third_party/unicode/tool/unicode_sync_script.dart

```diff
@@ -179,33 +142,32 @@ abstract class PropertiesSyncer {
 
   Future<void> perform() async {
     final List<String> lines = await File(_src).readAsLines();
-    final List<String> header = extractHeader(lines);
     final PropertyCollection data =
         PropertyCollection.fromLines(lines, defaultProperty);
 
-    final String output = template(header, data);
+    final String output = template(data);
 
     if (_dryRun) {
+      print('=' * 80);
+      print(path.relative(_src, from: unicodeRoot));
+      print('=' * 80);
```

Comment:
Bonus points if we could have the dryrun to also compare the contents of the generated Dart files and issue a warning if it's out of sync? If it also set the exit code to a non-zero value, we could use this script as a test that verifies that the data is consistent.